examples/workers/basic-module/index.ts

```diff
@@ -0,0 +1,20 @@
+import { Router } from 'worktop';
+import { listen } from 'worktop/modules';
+
+// binding: ENV variable
+declare const FALLBACK: string;
+
+const API = new Router();
+
+API.add('GET', '/greet/:name?', (req, res) => {
+	res.end(`Hello via Module, ${req.params.name || FALLBACK}!`);
+});
+
+API.add('GET', '/', (req, res) => {
+	const command = `$ curl https://${req.hostname}/greet/lukeed`;
+
+	res.setHeader('Cache-Control', 'public,max-age=60');
+	res.end(`Howdy Module Worker~! Please greet yourself; for example:\n\n  ${command}\n`);
+});
+
+export default listen(API.run);
```

Comment:
This `listen` comes from `worktop/modules`, which converts the ServiceWorker-based `API.run` handler and prepares it for a Module Worker `fetch` signature.

**Important:** Must have `export default` here, as the `listen` is returning the `{ fetch }` worker object.

Reply:
it's named `listen` to stay consistent with `worktop` and `worktop/cache` methods – all ServiceWorker-facing "start the app" methods are called `listen`